requirements.txt

```diff
@@ -1,65 +1,14 @@
-alabaster==0.7.12
-attrs==19.3.0
-Babel==2.8.0
-cachetools==4.1.0
-certifi==2020.4.5.1
-cffi==1.14.0
-chardet==3.0.4
-coverage==5.1
-cryptography==3.3.2
-docutils==0.16
-google-api-core==1.17.0
-google-api-python-client==1.8.3
-google-auth==1.15.0
-google-auth-httplib2==0.0.3
-google-auth-oauthlib==0.4.1
-googleapis-common-protos==1.51.0
-httplib2==0.19.0
-idna==2.9
-imagesize==1.2.0
-importlib-metadata==1.6.0
-Jinja2==2.11.3
-MarkupSafe==1.1.1
-more-itertools==8.3.0
-mypy==0.770
-mypy-extensions==0.4.3
-oauthlib==3.1.0
-packaging==20.4
-pip==20.1.1
-pluggy==0.13.1
-protobuf==3.11.3
-py==1.8.1
-pyasn1==0.4.8
-pyasn1-modules==0.2.8
-pycparser==2.20
-Pygments==2.6.1
-PyJWT==1.7.1
-pyparsing==2.4.7
-pytest==5.4.2
-python-dateutil==2.8.1
-pytz==2020.1
-requests==2.23.0
-requests-futures==1.0.0
-requests-mock==1.8.0
-requests-oauthlib==1.3.0
-requests-toolbelt==0.9.1
+cryptography>=3.3.2,<4
+google-api-core>=1.17.0,<2
+google-api-python-client>=1.12.8,<2
+google-auth>=1.28.0,<2
+google-auth-httplib2>=0.1.0,<1
+google-auth-oauthlib>=0.4.3,<1
+googleapis-common-protos>=1.53.0,<2
+python-dateutil>=2.8.1,<3
+pytz>=2020.1
 git+https://github.com/WIPACrepo/rest-tools@v1.1.0#egg=rest_tools
-rsa==4.1
-setuptools==46.4.0
-six==1.15.0
-snowballstemmer==2.0.0
-Sphinx==3.0.3
-sphinxcontrib-applehelp==1.0.2
-sphinxcontrib-devhelp==1.0.2
-sphinxcontrib-htmlhelp==1.0.3
-sphinxcontrib-jsmath==1.0.1
-sphinxcontrib-qthelp==1.0.3
-sphinxcontrib-serializinghtml==1.1.4
-tornado==6.0.4
-typed-ast==1.4.1
-typing-extensions==3.7.4.2
-uritemplate==3.0.1
-urllib3==1.25.9
-wcwidth==0.1.9
-wheel==0.34.2
-zipp==3.1.0
+# helpers
```

Comment:
@dsschult I'm surprised setuptools doesn't freak out because this isn't a package

Reply:
`#` is a comment, even in this file.  It's part of the pip standard.

Reply:
I mean it doesn't have a problem with adding that commented-line to the `install_requires` list: https://github.com/WIPACrepo/zoom-meeting-list/blob/master/setup.py#L51